flink-ml-dist/pom.xml

```diff
@@ -19,14 +19,17 @@ under the License.
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <statefun-flink-core.version>3.2.0</statefun-flink-core.version>
+    </properties>
```

Comment:
nit: Let's move this property to the parent pom.xml and reuse this property in flink-ml-dist and flink-ml-iteration pom files.

Besides, `statefun.version` might be enough while more concise.

Reply:
sure. Will do shortly